text/0000-devtools-protocol.md

```diff
@@ -24,44 +24,37 @@ See the current protocol here: https://xstate.js.org/docs/packages/xstate-inspec
 `globalThis.__xstate__` is a global `XStateDevInterface` object:
 
 ```ts
-interface InspectedEventObject {
-  name: string; // Event type
-  data: AnyEventObject; // The actual event object
-  origin?: string; // Session ID
-  destination: string; // Session ID
-  createdAt: number; // Timestamp
-}
-
 interface InspectedActorObject {
```

Comment:
This concern seemed to warrant its own comment. A big difference between this model and the Workflow is that notion of a `system`. A Workflow can be an actor that is also the origin of several other workflows, making the `system` essentially a parent Workflow. We don't seem to have that concept here, and I think for crawling events, we may want to add it.

Reply:
Added `systemId`